src/components/molecules/NavigationIcons.vue

```diff
@@ -1,8 +1,8 @@
 <template>
 	<div class="icons">
-		<Cart/>
+		<Cart :cart="icons.cart"/>
```

Comment:
why prop for cart is cart? use undrstandable naming for prop that expected to be passed like `cartIcon`